tendrl/node_agent/message/handler.py

```diff
@@ -0,0 +1,53 @@
+from gevent import socket
+from gevent.socket import error as socket_error
+from gevent.socket import timeout as socket_timeout
+import gevent.event
+import gevent.greenlet
+from gevent.server import StreamServer
+from io import BlockingIOError
+import os
+from tendrl.commons.message import Message
+from tendrl.node_agent.message.logger import Logger
+
+
+RECEIVE_DATA_SIZE = 4096
+
+class MessageHandler(gevent.greenlet.Greenlet):
```

Comment:
@nnDarshan if the user runs node_agent as service then how they will know where exception occurred, this problem also in event.py in commons, if socket fails i don't know what to do, if socket fails i am raising an exception in event.py but try catch is required for all the places where log message is created. we can handle error there but problem is for alerting anmol wants that exception in collecd.